src/ExampleData/disastersExampleData.js

```diff
@@ -0,0 +1,11 @@
+import moment from 'moment';
+import disasterImage from 'Images/Disasters/Assets/wolves.png';
+
+const disastersExapmleData = {
+	id: '1',
+	title: 'wolves',
+	disasterImage,
```

Comment:
please use a reference to the image, with a dictionary at Disaster's folder.
please fix the same issue at `characterInfoExampleData.js`